packages/nuget-exe/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nuget-exe",
+  "version": "5.8.0",
+  "description": "Repackaged NuGet.exe binary",
+  "main": "nuget-5.8.0.exe",
+  "repository": "https://github.com/microsoft/react-native-windows",
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean"
+  },
+  "devDependencies": {
+    "just-scripts": "^0.44.7"
```

Comment:
Our repo is using a newer version of this now I believe. The version should be updated to match what is currently in the repo.

Reply:
Upgrading to version `^1.3.3` as in the `vnext` package manifest.